q2_deblur/_denoise.py

```diff
@@ -147,13 +147,12 @@ def _denoise_helper(
                                              pd.DataFrame):
     _check_inputs(**locals())
     with tempfile.TemporaryDirectory() as tmp:
+        df = demultiplexed_seqs.manifest.view(pd.DataFrame)
+        for id in df.index:
+            if '_' in id:
+                raise ValueError("Sample IDs may not contain underscores, the "
```

Comment:
Just realized it is probably worth making the distinction that it is `deblur` that has this requirement, not QIIME 2. Can you change to something like `deblur cannot operate on sample IDs that contain underscores...`?